docs/source/install-django-project.rst

```diff
@@ -22,161 +22,120 @@ downloading data as a researcher.
    PR to the lookit-api repo to update the documentation! For notes on Linux installation,
    there may be helpful information in a `previous version of invoke tasks.py <https://github.com/lookit/lookit-api/blob/d1b8c9b43cb7d7bda7cdbe5958236d99af42341d/tasks.py>`__.
 
-Django
-~~~~~~
-
-These instructions will get walk you through the installation and running of the Django's local envionment.  
-
-#. Install `Homebrew <https://brew.sh/>`__.  Run update and upgrade:
 
-   .. code-block:: shell
+Prerequisites
+~~~~~~~~~~~~~
 
-      brew update && brew upgrade
+Before you can begin setting your local environment, we'll need to install a couple of things.
 
-#. Install `Docker <https://docs.docker.com/docker-for-mac/install/>`__ and make sure it's running.
+#. Install `mkcert <https://github.com/FiloSottile/mkcert>`__.
 
-#. Install Git and Poetry:
+#. Install `Docker Desktop <https://www.docker.com/products/docker-desktop/>`__.
 
-   .. code-block:: shell
+Local Environment
+~~~~~~~~~~~~~~~~~
 
-      brew install git poetry
+Now the Prerequisites have been installed and Docker is running we can setup the local environment.
 
-#. Change directory to where you want the project to live.  Next, clone repo and change directory.
+#. Clone lookit-api and change directory:
 
    .. code-block:: shell
 
       git clone git@github.com:lookit/lookit-api.git
       cd lookit-api
 
-#. Set Poetry's Python and install Python packages:
+#. Create environment file:
 
    .. code-block:: shell
+      
+      cp env_dist .env
 
-      poetry env use python3.9
-      poetry run pip -U pip wheel setuptools
-      poetry install
-
-#. Create database and run setup script:
+#. Make local CA and certificates for HTTPS:
 
    .. code-block:: shell
 
-      poetry run invoke create-db setup
-
-   This will create a local .env file with environment variables for local development,
-   run the Django application's database migrations ("catching up" on changes to the 
-   database structure) and create local SSL certificates. If you're curious about what 
-   exactly is happening during this step, or run into any problems, you can reference the 
-   file `tasks.py <https://github.com/lookit/lookit-api/blob/develop/tasks.py>`__.
+      make local-certs
 
-#. Run local environment server:
+#. Run DB migrations and add entry into Site table:
 
    .. code-block:: shell
 
-      poetry run ./manage.py runserver
+      make migrate
+      make site
 
-#. Navigate to local environment at http://localhost:8000.
-
-Rabbitmq
-~~~~~~~~
+#. Celery worker will need some permissions setup to run correctly.  To setup these permissions, we'll need to first start the Docker services:
 
-These instructions will have you create and start a RabbitMQ image in Docker.
-
-Install and start rabbitmq via docker:
+   .. code-block:: shell
 
-.. code-block:: shell
+      make serve
```

Comment:
Some commands (like this one, I think) generate a lot of output in the terminal. It might be helpful to add screenshots or text blocks showing this output and what it should look like if it has finished successfully. (If you agree then we could add this later, if you don't want to do it for this PR.)